chkconfig-cos7-s390x/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: chkconfig-cos7-s390x
+  version: 1.7.6
+
+source:
+  - url: {{ clefos7_url }}/s390x/chkconfig-1.7.6-1.el7.s390x.rpm
+    sha256: cbda2cf28daf41a6a6adc0395cd6fd462d921c4a1b258668bd43a172335f62a8
+    no_hoist: true
+    folder: binary
+  - path: ../common_files
+
+build:
+  number: 2
```

Comment:
```suggestion
  number: 1
```

Reply:
@cdeepali : I have addressed the review comments in latest commit. Thanks!

Reply:
Thanks @pradghos.